ios/RNSScreen.mm

```diff
@@ -889,8 +889,8 @@ - (void)updateFormSheetPresentationStyle
     sheet.delegate = self;
 #if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && defined(__IPHONE_16_0) && \
     __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_16_0
-    if (_sheetAllowedDetents.count > 0) {
-      if (@available(iOS 16.0, *)) {
+    if (@available(iOS 16.0, *)) {
+      if (_sheetAllowedDetents.count > 0) {
```

Comment:
Why is this swap done?

Reply:
_sheetAllowedDetents.count > 0 is true in most cases. Nothing would happen on iOS < 16 as such case isn't handled inside the condition. After swapping them both versions work correctly

Reply:
okay, thanks!